app/javascript/packs/ssn-field.ts

```diff
@@ -1,14 +1,13 @@
 import Cleave from 'cleave.js';
 
-/* eslint-disable no-new */
 function formatSSNFieldAndLimitLength() {
-  const inputs = document.querySelectorAll('input.ssn-toggle[type="password"]');
+  const inputs = document.querySelectorAll<HTMLInputElement>('input.ssn-toggle[type="password"]');
 
   if (inputs) {
```

Comment:
Future refactoring opportunity: This condition is unnecessary since the return value of `querySelectorAll` will always be truthy.